components/suggest/src/db.rs

```diff
@@ -1022,6 +1023,22 @@ impl<'a> SuggestDao<'a> {
             "DELETE FROM prefix_keywords WHERE suggestion_id IN (SELECT id from suggestions WHERE record_id = :record_id)",
             named_params! { ":record_id": record_id.as_str() },
         )?;
+        #[cfg(feature = "fakespot")]
+        {
+            self.scope.err_if_interrupted()?;
+            // Delete FTS data for this record, which counter-intuitively requires an INSERT
```

Comment:
o.O That's really odd. Regardless, we might want to benchmark the FTS deletion bits to better understand its performance characteristics.